content/en/universal_service_monitoring/_index.md

```diff
@@ -46,10 +47,10 @@ If you have feedback about what platforms and protocols you'd like to see suppor
 
 ### Prerequisites
 
-- Datadog Agent 7.40 or higher is installed alongside your service. Installing a tracing library is _not_ required.
+- Datadog Agent 6.40/7.40 or higher is installed alongside your service. Installing a tracing library is _not_ required.
```

Comment:
Do we need this? It's duplicated with whats above and that section seems to be clearer and easier to read in **Supported versions and compatibility**